src/helpers/utilities.ts

```diff
@@ -253,6 +254,15 @@ export const handleSignificantDecimals = (
     new BigNumber(value).toFixed(decimals)
   ).toFixed();
   const resultBN = new BigNumber(result);
+  if (abbreviate) {
```

Comment:
adds formatting option for abbreviations like 1100 => 1.1k, 12345678 => 12.3m.

Reply:
No chance of trillions? :P